### DISCUSSION TITLE
curated list of themes

### DISCUSSION THREAD

**Discussion by trusktr** [heremaps/harp.gl#2378] [General]
Is there a list of themes somewhere, besides the themes in the main `@here/harp-map-theme` package? I wonder if people have created any themes and host them somewhere.

If there's no list, we can start one here. :)

**Comment by zbigg** [comment]
I think the only one created long time ago (and probably needing adaptations) was "Google based" standard theme adapted by @Astrak: https://github.com/Astrak/harp-standard-theme

I wasn't aware of any _other_ open theme, but i'm not involved for quite long time.
